src/shapes/IText/__snapshots__/ITextBehavior.test.ts.snap

```diff
@@ -2,63 +2,17 @@
 
 exports[`IText cursor animation snapshot exiting from a canvas will abort animation 1`] = `
 [
+  "0.000",
   "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
+  "0.996",
+  "0.986",
+  "0.969",
+  "0.944",
 ]
 `;
 
 exports[`IText cursor animation snapshot initDelayedCursor false - with delay 1`] = `
 [
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
-  "1.000",
```

Comment:
this first animation of 600ms of just full opacity is gone, the cursor start blinking sooner when there is no delay